server/filestore.go

```diff
@@ -2634,7 +2638,6 @@ func (fs *fileStore) ConsumerStore(name string, cfg *ConsumerConfig) (ConsumerSt
 		name: name,
 		odir: odir,
 		ifn:  path.Join(odir, consumerState),
-		fch:  make(chan struct{}),
```

Comment:
But remove from `consumerFileStore` struct too ;-)

Reply:
Same for qch btw... unless I am missing something these 2 did not seem to be referenced anywhere.